apps/docs/app/ui/components/componentExample/ComponentPreviewWrapper.tsx

```diff
@@ -44,11 +44,11 @@ const ComponentPreviewWrapper = memo(({ preview, toggleButton, minHeight = "13re
                     colorMode={localColorMode}
                 />
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/65447172-b7e9-45e4-b962-4b1133c2d3d6)
I would show space-between as it's clearer

Reply:
ok I could change it to this